### ISSUE TITLE
Support passing credentials file with systemd directive `OpenFile=`

### ISSUE THREAD

**Issue by eriksjolund** [yandex-cloud/geesefs#75]
The systemd directive [`OpenFile=`](https://www.freedesktop.org/software/systemd/man/systemd.service.html#OpenFile=) was introduced in __systemd 253__ (released 15 February 2023).

Support passing the credentials file with the systemd directive `OpenFile=`

This would be useful when running a systemd system service that makes use of 

```
User=testuser
```
because _testuser_ would not need read permission to the credentials file.

In other words, it would be more secure to use `OpenFile=`  than   __--shared-config__ in such a situation.
